### DISCUSSION TITLE
Import SDR# Bandplans and change radio mode automatically when tuned onto them?

### DISCUSSION THREAD

**Discussion by Voyager-III** [AlexandreRouma/SDRPlusPlus#550] [Ideas]
Would it be possible to add some sort of function that can import or use SDR# Bandplans, I have seen a few third party ones people have made and this would be great, And an option to automatically change radio modes when tuning on to the bandplan like SDR# Thanks

**Comment by AlexandreRouma** [comment]
I don't want to support multiple formats for band plan.
I suggest someone write a conversion script instead
